plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/utils/GeneralUtils.java

```diff
@@ -891,9 +891,6 @@ public static UUID getMixedEndianUUIDFromBytes(byte[] bytes) {
     }
 
     public static void validateResourceName(String name) throws DBException {
-        if (!DBWorkbench.isDistributed() && !DBWorkbench.getPlatform().getApplication().isMultiuser()) {
```

Comment:
Cannot just remove it. This function is used in many too places.

Reply:
The function was used in only one place, so I moved the condition there.